build.gradle

```diff
@@ -103,7 +105,7 @@ runPluginVerifier {
     // Rather arbitrarily chosen first releases of last three year.
     // Product version in use statistics say last 3 major releases. It does not define "major release" though but
     // but educated guess is that `year.version` is what is considered major.
-    ideVersions = ["2020.3.2", "2021.3.3", "2022.1"]
+    ideVersions = ["2020.3.4", "2021.3.3", "2022.2.3"]
```

Comment:
These are only used by the Plugin Verifier. IMO, not relevant to the fix but OK to have.